app/views/editorial/nodes/show.html.haml

```diff
@@ -1,6 +1,3 @@
-= content_for :title do
-  = node.name
-
```

Comment:
Maybe you could add a comment explaining why this file doesn't have a `content_for :title`? Then if someone comes by later and notices it's not there, they won't add it by accident. And it we change the way previews are handled, we know we might need to add it back.